pages/dashboard.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Head from 'next/head';
+
+import Dashboard from '../components/Dashboard';
+import styles from './dashboard.scss';
+
+import { fetchConferenceData } from '../api/client';
+
+const DashboardPage = ({ events, sponsorImages }) => (
+  <div>
+    <Head>
+      <link
+        rel="stylesheet"
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
+        integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
+        crossOrigin="anonymous"
+      />
+    </Head>
+    <div className={styles.videoContainer}>
+      <video autoPlay muted loop id="video-background">
+        <source
+          src="../static/assets/dashboard_background.mp4"
+          type="video/mp4"
+        />
+      </video>
+      <Dashboard events={events} sponsorImages={sponsorImages} />
+    </div>
+  </div>
+);
+
+DashboardPage.getInitialProps = async () => {

```

Comment:
I choose NextJS when first building the site because I thought I would have a server to host the site on and our conference would have SEO benefits for being SSR. I wasn't told RP was going to be hosted using s3/cloudfront until a lot of the project was set up and after I had already briefed the team on how to use Next and our Next project structure, so we're still using Next... oof. 

Because we are using Next static HTML builds (https://nextjs.org/docs#usage), getInitialProps is run on export time which means that if you updated the JSON files, your site would always be 1 off, meaning you'd be rendering the previous version of the JSON files.

getInitialProps should not be used for fetching JSON files because of this. Refer to `/pages/index.js` to how we're loading json data.

Like I said, the site is hacky af.